addons/bestja_detailed_reports/__openerp__.py

```diff
@@ -19,9 +19,9 @@
     ],
     'data': [
         'views/project.xml',
+        'views/wizards.xml',
         'views/detailed_reports.xml',
         'views/detailed_reports_summary.xml',
-        'views/wizards.xml',
```

Comment:
Is this a random change or does it mean something? If the latter, then a comment should be included. If the former, the change should be removed.